src/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import PropTypes from 'prop-types';
+import GithubSvg from './about-us/githubSvg';
+
+export default function Footer(props){
+  const enigmaBay = <a href='https://github.com/EnigmaBay' className='themed-hyperlink' data-theme={props.dataTheme}>EnigmaBay</a>;
+  const mit = <a href='https://opensource.org/licenses/MIT' className='themed-hyperlink' data-theme={props.dataTheme}>MIT License</a>;
+  return (
+    <Card data-theme={props.dataTheme} className='text-center themed-footer'>
```

Comment:
Following the DevBio pattern, add an 'mb-3' to the className (or a smaller number than 3) to force a margin below the Card output.

Reply:
Actually, there's probably a better way.

Reply:
If a div is used then a p-1 and rounded-2 can be applied to get a similar effect and reduce the line-height between text entries.

Reply:
Are you suggesting using a div rather than a Card as the container for the Footer?

Reply:
I discovered it was the padding on the "themed-text" class causing the appearance of too much line height, so I created a new CSS class to style the text in the footer. However, since there is no real reason to use the Card component here, I am leaving div as the parent class.